templates/coredata/swift3.stencil

```diff
@@ -1,10 +1,16 @@
 // swiftlint:disable all
 // Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
 
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable sorted_imports
 import CoreData
 import Foundation
+{% for import in param.extraImports %}
+import {{ import }}
+{% empty %}
+{% if param.extraImports %}import {{ param.extraImports }}{% endif %}
```

Comment:
Maybe a `{# If extraImports is a single String instead of an array, for considers it empty but we still have to check if there's a single String value #}` comment there to explain that strangeness you're talking about in your PR comment ?